.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
+  "tabWidth": 2,
```

Comment:
This doesn't want to be changed, it's the same across all projects and this will alert it for this project only. For those of us working across multiple projects it'll be a bit disconcerting. You should be able to change locally in the IDE.

Reply:
Good point 👍